### PR TITLE
Login

### PR DESCRIPTION
**Pull request by alegrimminck**
Context
In the teacherBook app we need a Login for the teacher/user that needs to be responsive

What did I do?
I created the login page for the teacher and added the fonts, colors and sizes of text from the figma desgin, specifically i added:
- responsive login page
- generic Input component
- generic Icon component
- fonts, colors and text sizes in tailwind.config

#### Extras
Ideally Gifs for the feature or images in case it's needed
login

### CODE REVIEW

File: src/assets/desktop-bottom.svg
Comment (yurehn) [inline src/assets/desktop-bottom.svg:1]: remove v 1.0.0 from svg

Comment (alegrimminck) [reply src/assets/desktop-bottom.svg:1]: Nice catch, done! :+1:

**Review by yurehn — APPROVED** [review]

File: src/components/Molecules/Icon.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { IconContext } from "react-icons";
+
+export default function Icon({ icon, color = "text-black", size = "16px" }) {
+  const Icon = icon;
```
Comment (javonunezz) [inline src/components/Molecules/Icon.jsx]: Revisar posibilidad de cambiar el nombre de la variable "Icon", ya que la funcion donde se declara tiene el mismo nombre

File: src/components/Molecules/Icon.jsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { IconContext } from "react-icons";
+
+export default function Icon({ icon, color = "text-black", size = "16px" }) {
```
Comment (javonunezz) [inline src/components/Molecules/Icon.jsx]: Revisar formato de export para todos los componentes, para que todos sean iguales

File: src/components/Molecules/Icon.jsx (hunk 1 above)
Comment (alegrimminck) [reply src/components/Molecules/Icon.jsx]: Listo, cambiado a iconComponent y ahora se pasa directamente el componente de react-icons :+1:

File: src/components/Molecules/Icon.jsx (hunk 2 above)
Comment (alegrimminck) [reply src/components/Molecules/Icon.jsx]: Cambié todos los que agregué a `export const FunctionName = () => {}` :+1:

**Pull request merged**